src/core/src/bootstrap/EnvLayer.py

```diff
@@ -313,7 +319,12 @@ def read_with_retry(self, file_path_or_handle):
                         if i < Constants.MAX_FILE_OPERATION_RETRY_COUNT - 1:
                             time.sleep(i + 1)
                         else:
-                            raise Exception("Unable to read from {0} (retries exhausted). Error: {1}.".format(str(file_path_or_handle), repr(error)))
+                            error_message = "Unable to read from {0} (retries exhausted). Error: {1}.".format(str(file_path_or_handle), repr(error))
```

Comment:
Same comment about logging format and inclusion of the additional variable.

Reply:
Resolved